### DISCUSSION TITLE
still being maintained and updated?  if not, can I help?

### DISCUSSION THREAD

**Discussion by mpstaton** [marp-team/marp#496] [Q&A] [answered]
I have spent almost all my adult life presenting. Sales, talks, video, team leadership, whatever. I still spend most of my life presenting.  I've been realizing over the last few years that almost all my time goes to "noodling" in Keynote, Powerpoint, whatever, and I want a DRY way of creating presentations that stays up to date and I can deploy on our marketing site.  Thanks, much love.  Also love the marp spelling, reminds me of Hot Fuzz.

**Comment by yhatt** [comment, marked as answer]
> still being maintained and updated?

Sure. Marp ecosytstem is still maintained. See also #401 :)

Having said that, it's true that I now have less time to devote to Marp and some open-sourced projects than before. We are always welcome to contribute to our projects.
https://github.com/marp-team/.github/blob/master/CONTRIBUTING.md
